exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# Instructions append
+
+  
```

Comment:
```suggestion
## Dates and Times in Python

This exercise explores objects from Python's `datetime` module:

```

Reply:
Thought there should be a little preamble here,  just to focus the student in on the links.